frontend/src/components/form/CheckBox/CheckBox.tsx

```diff
@@ -1,9 +1,20 @@
-import PropTypes from "prop-types";
 import React from "react";
+
 import Label from "../../@common/Label/Label";
 import styles from "./CheckBox.module.css";
 
-const CheckBox = ({ label, name, checked, onChange, required, ...props }) => {
+type CheckBoxProps = React.InputHTMLAttributes<HTMLInputElement> & {
+  label?: string;
```

Comment:
r: 디폴트 파라미터이기 때문에 옵셔널 제거해도 될 것 같습니다.

```suggestion
  label: string;
```

Reply:
음...디폴트 파라미터로 할당하기 때문에 optional로 지정하는게 맞지 않나요? 👀

Reply:
제가 잘못 생각했네요. CheckBox 컴포넌트 내에서 옵셔널을 제거해도 타입 에러가 나오지 않길래 제거해도 된다고 생각했습니다. 옵셔널 제거 시 사용처에서 label 할당하지 않으면 에러가 출력되네요.